pkg/controller/registrysyncer/registrysyncer.go

```diff
@@ -235,7 +235,8 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, log *
 		log.Debug("creating imageStreamImport")
 		if err := createAndCheckStatus(ctx, client, imageStreamImport); err != nil {
 			controllerutil.CountImportResult(ControllerName, clusterName, req.Namespace, imageStreamName, false)
-			return fmt.Errorf("failed to create and check the status for imageStreamImport on cluster %s %v : %w", clusterName, imageStreamImport, err)
+			//no return of the error to avoid staying in the queue forever
+			log.WithError(err).Errorf("failed to create and check the status for imageStreamImport for tag %s of %s in namespace %s on cluster %s", imageTag, imageStreamImport.Name, imageStreamImport.Namespace, clusterName)
```

Comment:
This kind of defeats the purpose of using a controller in the first place, as we will never retry on transient errors. Also, metrics are wrong

Reply:
I think what we should be doing is if the src import had any errors and skip the ones where it had errors

Reply:
Implemented as discussed in slack: no importing back to where the image came from.

Reply:
>metrics are wrong

Can you elaborate?